.gitignore

```diff
@@ -1,152 +1,5 @@
 # Byte-compiled / optimized / DLL files
 __pycache__/
-*.py[cod]
-*$py.class
-
-# Folders
-xarray_test/*
-data/*
-
-# C extensions
-*.so
-
-# Distribution / packaging
-.Python
-build/
-develop-eggs/
-dist/
-downloads/
-eggs/
-.eggs/
-lib/
-lib64/
-parts/
-sdist/
-var/
-wheels/
```

Comment:
these might all be useful :)

Reply:
might or are? I'd prefer to keep this simple, but if there are specific issues you ran into without these, happy to keep them